vsts/test-release-nuget.yaml

```diff
@@ -103,8 +103,7 @@ jobs:
           IOTHUB_X509_PFX_CERTIFICATE: $(IOTHUB-X509-PFX-CERTIFICATE)
           DPS_IDSCOPE: $(DPS-IDSCOPE)
           DPS_GLOBALDEVICEENDPOINT: $(DPS-GLOBALDEVICEENDPOINT)
-          DPS_INDIVIDUALX509_PFX_CERTIFICATE: $(DPS-INDIVIDUALX509-PFX-CERTIFICATE)
```

Comment:
Why isn't the individual cert needed anymore? Is the name with "group" accurate?

Reply:
We're generating both self-signed and CA signed certificates on-demand. Therefore, we don't need to hardcode the certificate in keyvault and pull it from there anymore.

The keyword "group" there is supposed to indicate that the device identity is being validated through an intermediate certificate that has already been verified on DPS. This intermediate certificate can be used to sign a "group" of device identities, all of which chain up to the verified intermediate certificate.

Reply:
Hmm, I mistook it to be associated with group enrollments. Can we consider a rename to avoid the confusion?